### DISCUSSION TITLE
How to export vm file and import it with vmware ?

### DISCUSSION THREAD

**Discussion by Taylan146** [utmapp/UTM#4355] [UTM Help (macOS)] [answered]
Hey guys Ive got a quick question. I just setup openDCIM onto an Ubuntu vm using UTM. I used UTM because all the other commonly used programs do not work with the new M1 chip that my Mac uses. So by using UTM in addition to qemu it made it really easy for me to set all of that up but now to my problem. Can I export the Qcow file of my Ubuntu vm from UTM and somehow import it to vmWare as a .vmdk file ?

**Comment by ideologysec** [comment, marked as answer]
https://dannyda.com/2020/06/25/how-to-use-qemu-img-command-to-convert-between-vmdk-raw-qcow2-vdi-vhd-vhdx-formats-disk-images-qemu-img-create-snapshot-resize-etc/

use qemu-img to convert to vmdk.

short version: `qemu-img convert -O vmdk /data/source.qcow2 /data/output.vmdk`

**Reply by Taylan146** [reply]
Thank you very much this was exactly what I was looking for !
